src/actor.rs

```diff
@@ -230,13 +230,16 @@ impl ActorState {
 /// actor's communication channels (message handling).
 pub trait ActorContext: Sized {
     /// Immediately stop processing incoming messages and switch to a
-    /// `stopping` state
+    /// `stopping` state. This only affects actors that are currently
+    /// `running`. Future attempts to queue messages will fail.
     fn stop(&mut self);
 
-    /// Terminate actor execution
+    /// Terminate actor execution unconditionally. This sets the actor
+    /// into the `stopped` state. This causes future attempts to queue
+    /// messages to fail.
     fn terminate(&mut self);
 
-    /// Actor execution state
+    /// Get or retrieve the current Actor execution state.
```

Comment:
Sorry for causing misudnerstanding, I meant to use either `Get` or `Retrieve` as verb here instead. Not both of them 😄 
Choose what yo like more

Reply:
Haha the comment did confuse me, but I decided "nah, I'll let it pass, @DoumanAsh must have something in mind". :)